torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -523,8 +526,8 @@ PyTorchModelLoader::loadValue(const torch::jit::Value *value) {
   RETURN_ERR_IF_NOT(value->isCompleteTensor(),
                     glow::strFormat("Value %s must have CompleteTensor type.",
                                     value->debugNameBase().c_str()));
-  auto ptType = value->type()->cast<at::CompleteTensorType>();
-  auto glowType = ptTypeToGlowType(*ptType.get());
+  auto glowType = ptTypeToGlowType(
+      *DCHECK_NOTNULL(value->type()->cast<at::ProfiledTensorType>()));
```

Comment:
fyi: you could also use `value->type()->expect<at::ProfiledTensorType>()` which also asserts that the cast is successful.